testplan/testing/multitest/driver/base.py

```diff
@@ -261,8 +266,25 @@ def _install_target(self):
         raise NotImplementedError()
 
     def _install_files(self):
-        for template in self.cfg.install_files:
-            instantiate(template, self.context_input(), self._install_target())
+
+        for install_file in self.cfg.install_files:
+            if isinstance(install_file, basestring):
+                if not os.path.isfile(install_file):
+                    raise ValueError("{} is not a file".format(install_file))
+                instantiate(
+                    install_file, self.context_input(), self._install_target(),
+                )
+            elif isinstance(install_file, tuple):
+                if len(install_file) != 2:
+                    raise ValueError(
+                        "Expected a (source, destination) pair; got {}".format(
```

Comment:
Can this be changed to `"Expected the the source filepath, or a (sourcepath, destinationpath) tuple; got {}"` ?